janitor/functions.py

```diff
@@ -784,14 +801,34 @@ def add_column(df, colname: str, value):
         for the column name to be compatible with the Feather binary
         format (this is a useful thing to have).
     :param value: Either a single value, or a list/tuple of values.
+    : fill_remaining: If value is a tuple or list that is smaller than
```

Comment:
Extraneous space. 
```suggestion
    :fill_remaining: If value is a tuple or list that is smaller than
```

Reply:
Actually it should probably read `:param fill_remaining:`.

Reply:
I'll accept it for now to push the change, but will update in a later commit.